notification/store.go

```diff
@@ -297,6 +297,28 @@ func (db *DB) VerifyContactMethod(ctx context.Context, cmID string, code int) er
 	return nil
 }
 
+func GetMessageStatusState(lastStatus string, hasNextRetry bool) MessageState {
```

Comment:
Same as above, since it's a helper declaratively describe what it does and then make it private (lower case first letter).

```suggestion
func messageStateFromStatus(lastStatus string, hasNextRetry bool) MessageState {
```

Reply:
this helper function is also used in the contact method store. I changed the name but shouldn't it stay public?

Reply:
We should probably move the method that's accessing outgoing_messages into the notification store instead anyhow. We're only using the contact method to lookup the message ID.